security/guard_authentication.rst

```diff
@@ -108,10 +165,10 @@ This requires you to implement several methods::
                 return null;
             }
 
-            // if a User is returned, checkCredentials() is called
-            return $this->em->getRepository(User::class)
-                ->findOneBy(['apiToken' => $credentials])
-            ;
+            // The "username" in this case is the apiToken, see the key `property`
+            // of `your_db_provider` in `security.yaml`.
+            // If this returns a user, checkCredentials() is called next:
+            return $userProvider->loadUserByUsername($apiToken);
```

Comment:
Should be `$credentials` instead of `$apiToken`.

See https://github.com/symfony/symfony-docs/pull/14410

Reply:
Right :-)